mathics/builtin/image.py

```diff
@@ -1134,16 +1134,14 @@ def _image_pixels(matrix):
         return None
 
 
-class ImageCreate(Builtin):
-    def apply(self, array, evaluation):
-        '''ImageCreate[array_]'''
-        pixels = _image_pixels(array.to_python())
-        if pixels is not None:
-            shape = pixels.shape
-            is_rgb = (len(shape) == 3 and shape[2] == 3)
-            return Image(pixels.clip(0, 1), 'RGB' if is_rgb else 'Grayscale')
-        else:
-            return Symbol('$Aborted')
+class ImageQ(Builtin):
+    def apply_image(self, image, evaluation):
+        'ImageQ[image_Image]'
+        return Symbol('True')
+
+    def apply_no_image(self, array, evaluation):
+        'ImageQ[Image[array_]]'
+        return Symbol('False')
```

Comment:
What happens when the argument doesn't match either pattern? I think a better way would be to inherit from `Test` and implement the test as `isinstance(arg, Image)`